ws-nextjs-app/pages/[service]/send/[id]/send.stories.tsx

```diff
@@ -1,13 +1,29 @@
-import React from 'react';
+import React, { PropsWithChildren } from 'react';
 import PageLayoutWrapper from '#app/components/PageLayoutWrapper';
+import { NextRouter } from 'next/router';
+import { RouterContext } from 'next/dist/shared/lib/router-context.shared-runtime';
 import mundoFormFixture from '#data/mundo/send/test2qq3x8vt.json';
 import UGCPage from './UGCPageLayout';
+import { PageProps } from './types';
+
+const NextRouterWrapper = ({ children }: PropsWithChildren) => (
+  <RouterContext.Provider
+    value={{ query: { id: '123' } } as unknown as NextRouter}
+  >
+    {children}
+  </RouterContext.Provider>
+);
```

Comment:
This allows exposure of the `useRouter` hook and subsequently the `query` value. Storybook v7 onwards has first-class support for Next.js, so we should remove this once we upgrade it.